src/views/pacbio/PacbioRunIndex.vue

```diff
@@ -167,12 +167,12 @@ const fields = ref([
   - `value`: The value to be used for the filter.
   - `text`: The text to be displayed for the filter option.
 */
-const filterOptions = ref([
+const filterOptions = [
   { value: '', text: '' },
   { value: 'name', text: 'Name' },
   { value: 'state', text: 'State' },
   // Need to specify filters in json api resources if we want more filters
-])
+]
```

Comment:
I hope this doesn't have any implications on removing ref.

Reply:
I am not sure why but it looks like the ref was causing the problem @SHIV5T3R can you explain why?

Reply:
the ref was breaking the filtering as the filter was expecting an array and receiving a ref object instead.

Reply:
My initial solution was
```js
filterOptions.filter(({ value }) => value == filter_value.value)[0]?.wildcard &&
```

But `filterOptions` doesn't really need to be a reference, it shouldn't have any implications

Reply:
Ok, thanks for the explanation. 👍